en/core-api/e-wallet.md

```diff
@@ -1,76 +1,75 @@
 # Core API E-Wallet Integration
 <hr>
 
-?> This payment method is compatible with [QR Code Indonesian Standard (QRIS)](https://www.bi.go.id/id/ruang-media/siaran-pers/Pages/SP_216219.aspx), and can be paid with **any QRIS compatible e-wallet or banking app**.
+Midtrans facilitates its customers by *E-Wallet*. This payment method is compatible with [QR Code Indonesian Standard (QRIS)](https://www.bi.go.id/id/ruang-media/siaran-pers/Pages/SP_216219.aspx). Customers can make payments with GoPay, ShopeePay or any QRIS compatible e-wallet or banking app. 
 
-GoPay is an e-Wallet payment method by Gojek. Users will pay using the Gojek apps, or any QRIS compatible app. The user flow varies when using a web browser (on a computer or a tablet) compared to a SmartPhone:
+Gojek has introduced GoPay as an E-Wallet payment method. Customers can pay using the Gojek apps, or any QRIS compatible app.  The user flow is different for web browser (on a computer or a tablet) and smartphone. The user flows are given below.
```

Comment:
No it is not about `Gojek has introduced GoPay as an E-Wallet`, it is about a brief of what is `GoPay`, which is e-wallet provided by Gojek.